engine/apps/alerts/tasks/alert_group_verbose_name.py

```diff
@@ -0,0 +1,70 @@
+from django.db.models import Min
+
+from apps.alerts.incident_appearance.templaters import TemplateLoader
+from apps.alerts.tasks.task_logger import task_logger
+from common.custom_celery_tasks import shared_dedicated_queue_retry_task
+from common.jinja_templater import apply_jinja_template
+
+# BATCH_SIZE is how many alert groups will be processed per second (for every individual alert receive channel)
+BATCH_SIZE = 1000
+
+
+def batch_ids(queryset, cursor):
+    return list(queryset.filter(id__gt=cursor).order_by("id").values_list("id", flat=True)[:BATCH_SIZE])
+
+
+@shared_dedicated_queue_retry_task
+def update_verbose_name_for_alert_receive_channel(alert_receive_channel_pk):
+    from apps.alerts.models import AlertGroup
+
+    countdown = 0
+    cursor = 0
+    queryset = AlertGroup.all_objects.filter(channel_id=alert_receive_channel_pk)
+    ids = batch_ids(queryset, cursor)
+
+    while ids:
+        update_verbose_name.apply_async((alert_receive_channel_pk, ids), countdown=countdown)
+
+        cursor = ids[-1]
+        ids = batch_ids(queryset, cursor)
+        countdown += 1
+
+
+@shared_dedicated_queue_retry_task
+def update_verbose_name(alert_receive_channel_pk, alert_group_pks):
```

Comment:
It's silent in terms of logs. Will it be comfortable for release engineer to roll it out?

Reply:
Added some additional debug logging